### ISSUE TITLE
Adding a name attribute to any input works on WebAsm and Mobile but crashes on Desktop

### ISSUE THREAD

**Issue by RazvanMudura** [DioxusLabs/dioxus#1100]
**Problem**

Adding a name attribute to any input crashes the application on form submit.

**Steps To Reproduce**
1. Create any input with a name attribute
2. Try to use it on form submit 

**Expected behavior**
This should not crash on any platform, but crashes on Desktop, while it works on WebAsm and Mobile

**Environment:**
 - Dioxus version: 0.3.2
 - Rust version: 1.67
 EDIT: Tested on rust 1.70 and still same problem
 - OS info: Windows
 - App platform: Desktop

Example code
```rust
pub fn Content(cx: Scope) -> Element {
    cx.render(rsx! {
        main {
            form {
                prevent_default: "onsubmit",
                onsubmit: move |event| {
                    let name = event.values.clone();
                    println!("{:?}", name);
                },

                div {
                    label {
                        r#for: "explain",
                        "Explain"
                    }

                    input {
                        name: "explain",
                        r#type: "checkbox",
                        id: "explain",
                    }
                }


                button {
                    r#type: "submit",
                    "Start Activity"
                }
            }

        }
    })
}
```

**Comment by Demonthos** [comment]
It looks like this issue was fixed after the 0.3 release. If you switch to the git version of dioxus with:
```toml
dioxus = { git = "https://github.com/DioxusLabs/dioxus" }
dioxus-desktop = { git = "https://github.com/DioxusLabs/dioxus" }
```
The example above works as expected

**Comment by pascalbehmenburg** [comment]
This issue may be closed, as I can confirm that this indeed is not an issue anymore on dioxus desktop.
